config/locales/idv/fr.yml

```diff
@@ -161,8 +161,8 @@ fr:
       sessions:
         id_information_message: tel qu'il apparaît sur votre carte d'identité officielle
         id_information_subtitle: Informations d'identification
-        no_pii: N'utilisez pas de véritables données personnelles (il s'agit d'une
-          démonstration seulement)
+        no_pii: TRUQUÉ N'utilisez pas de véritables données personnelles (il s'agit
+          d'une TRUQUÉ démonstration seulement)
```

Comment:
Can we move the second `TRUQUÉ` to the end of the string, after the closing parenthesis?